### PR TITLE
Release stable 15.0.0

### PR DESCRIPTION
**Pull request by voxpelli**
Work on getting https://github.com/standard/eslint-config-standard/issues/208 done

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,6 +1,10 @@
-.github/
-test/
-tmp/
-.editorconfig
-tsconfig.json
-coverage
+*
```
Comment (Divlo) [inline .npmignore:1]: Could you explain why this change is needed? Are some files not included for `npm`?

Comment (voxpelli) [reply .npmignore:1]: It was about to push some `.vscode` folders of mine

Comment (voxpelli) [reply .npmignore:1]: As `.npmignore` overrides `.gitignore`, making it seems like you are safe, while you're not (Referenced my old tweet on the subject https://twitter.com/voxpelli/status/1181166507319005184)

Comment (Divlo) [reply .npmignore:1]: Okay thanks for the reference, I understand better the principle now.
We will need to not forget to update it if we add new files, we want to push on `npm`.

**Review by Divlo — APPROVED** [review]
